src/parser/jobs/rdm/Combos.js

```diff
@@ -0,0 +1,132 @@
+import CoreCombos from 'parser/core/modules/Combos'
+import ACTIONS from 'data/ACTIONS'
+import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import React from 'react'
+import {Plural, Trans} from '@lingui/react'
+
+export default class Combos extends CoreCombos {
+	// Overrides
+	static suggestionIcon = ACTIONS.ENCHANTED_REDOUBLEMENT.icon
+
+	_comboBreakers = []
+	_uncomboedGcds = []
+	//These actions are considered a combo DERP
+	_derpComboActions = [
+		ACTIONS.ENCHANTED_RIPOSTE.id,
+		ACTIONS.ENCHANTED_ZWERCHHAU.id,
+		ACTIONS.ENCHANTED_REDOUBLEMENT.id,
+	]
+	_severityDerpComboActions = {
+		1: SEVERITY.MINOR,
+		2: SEVERITY.MEDIUM,
+		3: SEVERITY.MAJOR,
+	}
+	//These actions mean you went in without enough mana
+	_notEnoughManaActions = [
+		ACTIONS.RIPOSTE.id,
+		ACTIONS.ZWERCHHAU.id,
+		ACTIONS.REDOUBLEMENT.id,
+	]
+	_severityNotEnoughManaActions = {
+		1: SEVERITY.MINOR,
+		2: SEVERITY.MEDIUM,
+		3: SEVERITY.MAJOR,
+	}
+	//Generics, not handled by the rest
+	_severityGenericActions = {
+		1: SEVERITY.MINOR,
+		2: SEVERITY.MEDIUM,
+		4: SEVERITY.MAJOR,
+	}
+
+	//Overrides
+	addJobSpecificSuggestions(comboBreakers, uncomboedGcds) {
+		this._comboBreakers = comboBreakers
+		this._uncomboedGcdCount = uncomboedGcds
+
+		//console.log('Output is Output!')
+		if (this._comboBreakers.length === 0 && this._uncomboedGcdCount.length === 0) {
+			//console.log('Output with no breakers!')
+			return false
+		}
+
+		//const panels = []
+		let derpComboCount = 0
+		let notEnoughManaCount = 0
+
+		if (this._comboBreakers.length > 0) {
+			//console.log('Breaker')
+			this._comboBreakers.map(breaker => {
+				// const util = require('util')
+				// console.log(util.inspect(breaker, {showHidden: true, depth: null}))
+				if (this._derpComboActions.includes(breaker.ability.guid)) {
+					//console.log(`${derpComboCount}: derpComboCount`)
+					derpComboCount++
+				}
+				if (this._notEnoughManaActions.includes(breaker.ability.guid)) {
+					//console.log(`${notEnoughManaCount}: notEnoughManaCount`)
+					notEnoughManaCount++
+				}
+			})
+		}
+
+		//Process Derped Combos
+		if (derpComboCount > 0) {
+			this.suggestions.add(new TieredSuggestion({
+				icon: ACTIONS.ENCHANTED_REDOUBLEMENT.icon,
+				why: <Plural
+					id="rdm.derpcombos.why"
```

Comment:
See the comment on `Gauge.js` about `Plural` tags. Also this is me nitpicking but translation ID conventions (should be something like `rdm.combos.suggestions.derpcombos.why`, and that applies to the other IDs in here too).

Reply:
I'll change my new Gauge suggestion ids with this change as well, however since I'm unsure how it would impact any existing translations I'm going to forgo changing the existing ones at this time.